tasks/prime.py

```diff
@@ -260,6 +261,31 @@ def update_mto_shipment_address(self):
         if success:
             self.replace_prime_data(PrimeObjects.MTO_SHIPMENT, mto_shipment, new_shipment)
 
+    @tag(PrimeObjects.MTO_AGENT.value, "updateMTOAgent")
+    @task
+    def update_mto_agent(self):
+        mto_shipment = self.get_random_data(
+            PrimeObjects.MTO_SHIPMENT
+        )  # this grabs response payload from the PrimeObjects.MTO_AGENT list when load testing runs
+        if not mto_shipment:
+            return  # can't run this task
+
+        payload = self.fake_request("/mto-shipments/{mtoShipmentID}/agents/{agentID}", "put")
+        mto_agent = mto_shipment["agents"][0]
+        headers = {"content-type": "application/json", "If-Match": mto_agent["eTag"]}
+        resp = self.client.put(
+            prime_path(f"mto-shipments/{mto_shipment['id']}/agents/{mto_agent['id']}"),
+            name=prime_path("mto-shipments/:mtoShipmentID/agents/mtoAgentID"),
```

Comment:
After talking with Abigail and Shimona, we decided to use { } curly braces to indicate the IDs in these names - so this line should look exactly like what you passed into the `self.fake_request` function on line 273